golang/ecommerce/user/controller/db.go

```diff
@@ -0,0 +1,20 @@
+package controller
+
+import (
+	"fmt"
+
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/mysql"
+)
+
+var db *gorm.DB
+var err error
+
+func Connect() *gorm.DB {
+	db, err = gorm.Open("mysql", "root:Test@123@tcp(127.0.0.1:3306)/ecommerce?charset=utf8&parseTime=True")
+	if err != nil {
+		fmt.Println("Connection Failed to Open")
+		panic(err)
```

Comment:
It's almost always a bad idea to panic - this function should return an error, and the caller (`main` function) should use `log.Fatalf` to log and exit.